subprojects/docs/src/docs/userguide/optimizing-performance/configuration_cache.adoc

```diff
@@ -581,6 +581,18 @@ Let your team know they can opt-in by, for example, enabling the configuration c
 Later on, when more workflows are working, you can flip this around.
 Enable the configuration cache by default, configure CI to disable it, and if required communicate the unsupported workflow(s) for which the configuration cache needs to be disabled.
 
+[[config_cache:adoption:reacting]]
+=== Reacting to the configuration cache in the build
+
+There are additional ways to streamline the migration process for the configuration cache compatibility.
```

Comment:
:thought_balloon: I'm not sure this introductory statement fits here, we're telling about a particular way of streamlining. I'd suggest to remove this paragraph.